### PR TITLE
Fixes #1674 Add nuget dir to PATH

### LINKED ISSUE #1674: Link to an explanation of adding nuget PATH variables.
The Windows installation section doesn't take into consideration whether the user understands PATH variables, so even simply a link to an explanation would be helpful.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6d00296b-ac62-3d6c-beea-b2042b63dc5b
* Version Independent ID: 587da972-dfde-88bc-9dc0-7c32dae3454b
* Content: [NuGet Command-Line Interface (CLI) Reference](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference#feedback)
* Content Source: [docs/reference/nuget-exe-CLI-Reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/nuget-exe-CLI-Reference.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

### PR DESCRIPTION
**Pull request by amoraitis**
Added steps to add the nuget.exe as an environment variable.

First suggested by @jerroldneal in #1674's comments

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[46f5d78](https://github.com/amoraitis/docs.microsoft.com-nuget/commits/46f5d78a1fc2e2059fcb1addd19673af1b8b3f53)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/install-cli.md](https://github.com/amoraitis/docs.microsoft.com-nuget/blob/master/docs/includes/install-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1713) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909291500260737-1713%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=B%2Fo7gRkFiOuMlFCSGmTfT%2Fug2%2BXjVBXzDbfyKC1jJAs%3D&st=2019-09-29T14%3A56%3A37Z&se=2019-10-30T15%3A01%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/install-cli.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
 1. Each download is the `nuget.exe` file directly. Instruct your browser to save the file to a folder of your choice. The file is *not* an installer; you won't see anything if you run it directly from the browser.
 1. Add the folder where you placed `nuget.exe` to your PATH environment variable to use the CLI tool from anywhere.
 
+    - Open the CMD
+    - Run the following command:
```
Comment (nkolev92) [inline docs/includes/install-cli.md]: I don't think this should be added. 

It's misusing a totally different feature unrelated to this.

Comment (amoraitis) [reply docs/includes/install-cli.md]: Do you want to abandon this PR? Or there is something you want to be updated?

Comment (zivkan) [reply docs/includes/install-cli.md]: I agree with @nkolev92 on this. The dotnet tools folder is managed by the dotnet cli, so the NuGet team should not encourage people to manually add/remove executables there.

If you'd like the docs to say that you can copy nuget.exe to a folder that's already on the path, then I'd be happy to approve the PR if you change it to say that. But I don't support using examples that encourage putting nuget.exe in other application's directories.

I appreciate that you took the time to write and submit this, but I don't think it should be included in the official docs as-is.

Comment (amoraitis) [reply docs/includes/install-cli.md]: @zivkan Yes, you are right. Would you like to add a reference(the exact command) to [setx](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx)? Or you just want to let people know that they have the option to move the `nuget.exe` to a folder that's already on the env path?

Comment (zivkan) [reply docs/includes/install-cli.md]: I think examples of how to use the product being documented is important, but I don't feel the same way about documenting other product/features. How to set environment variables is out of scope for NuGet, so I'd prefer to link to other docs that explain how to do that, but I'm unable to find docs on that on docs.microsoft.com.

Comment (amoraitis) [reply docs/includes/install-cli.md]: It seems to me that you don't like any change to the current version of the document. Consequently, I am closing this PR.

**Pull request closed without merging**